### DISCUSSION TITLE
Editable title & sync status

### DISCUSSION THREAD

**Discussion by maxkfranz** [cytoscape/cytoscape-explore#102] [Ideas]
Currently, the title is not editable in the UI.  There are several ways we could accommodate that.  Here are some examples:

Mac:  You click the title and get a tooltip where you can edit the file name and some metadata.  This approach would be useful if we have other things to edit than just the title in future (a link to the file manager, tags, citation info, etc.)

<img width="621" alt="Screen Shot 2021-11-22 at 10 48 21 AM" src="https://user-images.githubusercontent.com/989043/142892058-2114ea6d-b7fe-4b6f-8d48-39ff4a48b743.png">

Google Docs:  The title is just a text box that is styled as regular text by default.  It looks like a text box only on focus:

Without focus:

![Screen Shot 2021-11-22 at 10 44 28 AM](https://user-images.githubusercontent.com/989043/142891734-809c9469-dbd6-4984-b5bc-2f5980ebd91d.png)

With focus:

![Screen Shot 2021-11-22 at 10 44 32 AM](https://user-images.githubusercontent.com/989043/142891754-27e26ad4-b9ee-40a7-8ee7-561030077539.png)

Alongside the title, we also have sync status.  There are several approaches we could use for that, as well:

- Show the last saved time
- Change the text as things are being edited to something like 'Saving...'
- Always just show 'Edits saved' unless the user disconnected from wifi or an error occurred

What do you think?  Other alternatives?

**Comment by gbader** [comment]
Personally, I've always liked the Google docs implementation of this feature.
